Frontend/src/components/Table/Row.svelte

```diff
@@ -13,6 +14,42 @@
   };
 </script>
 
+<tr
+  on:click
+  style={`--rowHeight: ${rowHeight}px; ${
+    rowBackgroundColorFunction
+      ? "background-color: " + rowBackgroundColorFunction(item)
+      : ""
+  }`}>
+  {#each columns as col}
+    <td>
+      {#if col.isImageUrl}
+        {#if item[col.key] && displayValue(col, item) !== ""}
+          <img src={displayValue(col, item)} alt="item" />
+        {/if}
+      {:else if cellBackgroundColorFunction}
+        {#await cellBackgroundColorFunction(col, item)}
+          <div class="cell">
+            {displayValue(col, item)}
+          </div>
+        {:then color}
+          <div class="cell" style={color}>
```

Comment:
I think it would look better if we would style the entire td instead of only the div here.

Reply:
Mhh, I kind of like it as it is at the moment, it's really a word highlight in the traditional sense, as only the word is having a highlight background. but we can think about highlighint the entire cell

Reply:
Mh, I still think it would look better if we would color the entire cells. It would look cleaner and be more consistent with the other highlightings (for recent rentals / highlighted items or customers). But since you implemented this you can decide.

Reply:
I'm ok with entire cells!

Reply:
With cell I meant the td. Sorry that was confusing since the div has a class "cell".